src/DataDepsGenerators.jl

```diff
@@ -46,41 +80,51 @@ include("Figshare.jl")
 include("JSONLD/JSONLD.jl")
 
 function message(meta)
-    escape_multiline_string("""
-    Dataset: $(meta.fullname)
-    Website: $(meta.website)
-    $(meta.description)
-    """) |> strip
+    netString =  """
+    register(DataDep(
+        \"$(meta.shortname)\",
+        \"\"\""""
+    netString = format_meta(meta.website, netString, "Website")
+    netString = format_meta(format_authors(meta.author), netString, "Author", true)
+    netString = format_meta(meta.maintainer, netString, "Maintainer", true)
+    netString = format_meta(meta.license, netString, "License", true)
```

Comment:
license should be after all the date fields,
as sometimes it is very long.
So it should be the last of the "frontmatter" short fields.